USB_Servo_node:
  ros__parameters:
    serial_port: "/dev/serial/by-id/usb-Pololu_Corporation_Pololu_Mini_Maestro_12-Channel_USB_Servo_Controller_00460938-if00"
    # Tilt
    port0:
      min: 350
      max: 2500
      rom: 360
    # PAN
    port1:
      min: 350
      max: 2500
      rom: 360
    # GRIPPER
    port3:
      min: 500
      max: 2500
      rom: 180

    # Lights
    enable_pwm_pin: true
    pwm_frequency: 11700  # Frequency for PWM control